private/pkg/app/appcmd/appcmd.go

```diff
@@ -74,6 +74,9 @@ type Command struct {
 	// that precedes all other functionality, and which prints the version
 	// to stdout.
 	Version string
+	// Whether the command is the root command. This is currently used only
```

Comment:
This isn't needed and should be removed - we detect this by saying if there are sub-commands on the assumedly-root command you passed to run.

Reply:
Done